main.go

```diff
@@ -59,8 +59,10 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
+	var moduleChartDir string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	flag.StringVar(&moduleChartDir, "module-chart", "/module-chart", "The location of module chart.")
```

Comment:
right now we use a custom env configuration. I would suggest moving this module-chart path into env to stay consistent with this idea. But in general good idea 👌🏿 The rest of the flags were generated at the beginning of the history of this repo and can be removed in future

Reply:
I couldn't find place with env

Reply:
[Here](https://github.com/kyma-project/serverless-manager/blob/main/internal/state/new.go) is an example. Right now it's a simple `os.Getenv`, but if we would have more configurations like this I think we should create a separate config package that would keep the whole env configuration in one place

Reply:
I will unify it another PR